tests/test_call_history_phase_tools.py

```
@@ -0,0 +1,609 @@
+        def request(self, **kwargs):
+            url = kwargs.get("url", "")
+            if "/ok" in url:
+                return _MockResponse200('{"call":"A"}')
+            return _MockResponse500()
+        async def __aenter__(self): return self
+        async def __aexit__(self, *a): return False
+
+    monkeypatch.setattr(gw.aiohttp, "ClientSession", _MockSession)
+
+    tool = _make_webhook(url="http://example.invalid/{caller_number}", payload_template="{}")
+    ctx_a = _FakeContext(); ctx_a.caller_number = "ok"
```

Ruff (0.15.12)

[error] Multiple statements on one line (semicolon)

(E702)